auxure/settings.py

```diff
@@ -129,6 +129,7 @@
 
 
 REST_FRAMEWORK = {
-
-    'PAGE_SIZE': 3
```

Comment:
You removed the pagination size here and that did cause a conflict, pls set it back and use a trailing comma to avoid errors when you test. You should have something like this:

REST_FRAMEWORK = {

    'PAGE_SIZE': 3,
    'DEFAULT_PERMISSION_CLASSES': [
   'rest_framework.permissions.AllowAny',
]
}
Notice the comma after the 3. pls correct and make your pull request again.
Conflicts were on auxure/settings.py file because you removed the pagination size, and on store/admin.py because you removed the PerfumeAdmin that was used and replaced it with ProductAdmin. Please effect the corrections and create the pull request again